components/nspanel_lovelace/types.h

```diff
@@ -200,6 +200,7 @@ struct generic_type {
   static constexpr const char* unavailable = "unavailable";
   static constexpr const char* on = "on";
   static constexpr const char* off = "off";
+  static constexpr const char* empty = "";
```

Comment:
I don't think this is being used anywhere.. maybe remove it if not?

Reply:
This is used in the initialization of the tilt icons. If they are set to generic_type::disable this is shown as text on the nspanel. 
![nspanel_disabled_icons](https://github.com/olicooper/esphome-nspanel-lovelace-native/assets/13124077/9c9e9626-0154-45f4-bb6a-41ddd1efcb6c)

Reply:
No problem, thanks for looking!